db/schema.rb

```diff
@@ -615,6 +617,19 @@
     t.jsonb "settings", default: {}
   end
 
+  create_table "label_associations", force: :cascade do |t|
+    t.bigint "label_id", null: false
+    t.string "labelable_type", null: false
+    t.bigint "labelable_id", null: false
+    t.datetime "created_at", null: false
+    t.datetime "updated_at", null: false
+    t.index ["label_id"], name: "index_label_associations_on_label_id"
+    t.index ["labelable_type", "labelable_id"], name: "index_label_associations_on_contact", where: "((labelable_type)::text = 'Contact'::text)"
+    t.index ["labelable_type", "labelable_id"], name: "index_label_associations_on_conversation", where: "((labelable_type)::text = 'Conversation'::text)"
+    t.index ["labelable_type", "labelable_id"], name: "index_label_associations_on_labelable"
+    t.index ["labelable_type", "labelable_id"], name: "index_label_associations_on_labelable_type_and_labelable_id"
+  end
+
```

Comment:
Are the changes in schema.rb from the migrations in this PR alone?

Reply:
Oh these shouldn't be here, not sure how these came in, fixing it